src/layout/rect/iter.rs

```diff
@@ -38,8 +38,10 @@ impl Iterator for Rows {
 pub struct Columns {
     /// The `Rect` associated with the columns.
     pub rect: Rect,
-    /// The x coordinate of the column within the `Rect`.
+    /// The x coordinate of the column within the `Rect` when iterating forwards.
     pub current_column: u16,
+    /// The x coordinate of the column within the `Rect` when iterating backwards.
+    current_column_back: u16,
```

Comment:
I chose not to make `current_column_back` pub, as I don' think it needs to be.  Given these are not exposed externally I would suggest renaming `current_column` as `current_column_fwd` and making it non-pub.

Reply:
I don't recall why I made these pub in the first place - very likely by mistake rather than intention.
Changing this would be a breaking change, but one that very likely would be not seen by anyone.
I'd remove the pub altogether and slap a couple of lines in the docs.

Reply:
Done and added a section to the `BREAKING-CHANGES.md` document:

```
### Removed public field `[current_column|current_row]` from `layout::rect::[Columns|Rows]` ([#1358])

[#1358]: https://github.com/ratatui/ratatui/pull/1358

The `pub` modifier has been removed from fields `layout::rect::Columns::current_column` and 
`layout::rect::Rows::current_row`. These fields were not intended to be public and should not
have been accessed directly.
```

Reply:
I removed the pub on the rect fields too.